newm-chain-db/src/main/kotlin/io/newm/chain/database/repository/LedgerRepositoryImpl.kt

```diff
@@ -447,11 +447,17 @@ class LedgerRepositoryImpl : LedgerRepository {
                                 txOutput.elementToByteArray(0)
                             )
                         when (val assetsOutput = txOutput.elementAt(1)) {
-                            is CborInteger -> lovelace = assetsOutput.bigIntegerValue()
+                            is CborInteger -> {
+                                lovelace = assetsOutput.bigIntegerValue()
+                            }
+
```

Comment:
Sounds like you ran into a new ktlint rule requiring the curly braces everywhere? Sounds like another dumb rule. Maybe we could disable it, like we did for the one requiring everything at the right-side of `=` in a new line.